http/server.lua

```diff
@@ -23,6 +23,17 @@ local function sprintf(fmt, ...)
     return string.format(fmt, ...)
 end
 
+local function is_no_keepalive(disallowed_map, useragent)
+    if type(disallowed_map) ~= 'table' or useragent == nil then
+        return false
+    end
+    if disallowed_map[useragent] ~= nil then
```

Comment:
I think here should be "true" check.
I see on example:
```
keepalive_disable = {
          ['curl/7.68.0'] = true
      },
```

But what if I write:
```
keepalive_disable = {
          ['curl/7.68.0'] = false
      },
```
?

Reply:
Fixed.

```lua
--- a/http/server.lua
+++ b/http/server.lua
@@ -24,10 +24,10 @@ local function sprintf(fmt, ...)
 end
 
 local function is_no_keepalive(disallowed_map, useragent)
-    if type(disallowed_map) ~= 'table' or useragent == nil then
+    if useragent == nil then
         return false
     end
-    if disallowed_map[useragent] ~= nil then
+    if disallowed_map[useragent] == true then
         return true
     end
```